targets/BootStrapFullFrameworkMSBuild.proj

```diff
@@ -19,7 +19,7 @@
 
     <InstalledNuGetFiles Include="$(MSBuildExtensionsPath)\Microsoft\NuGet\*" />
 
-    <FreshlyBuiltBinaries Include="$(DeploymentDir)*.dll" />
+    <FreshlyBuiltBinaries Include="$(DeploymentDir)**\*.dll" />
```

Comment:
That's correct, it now needs the `**` to find the one level deeper satellite assemblies and to reconstruct the recursive dir path.